### ISSUE TITLE
Add Github "Fork this repo" ribbon

### ISSUE THREAD

**Issue by ChimeraCoder** [ChimeraCoder/pluto#12]
....sounds like a task for @rubinovitz or @exchgr! ;)

We want this to appear on the main page (`posts.tmpl`) but not on any other page.

If I add it directly to `posts.tmpl`, it looks like this:

![2013-11-27-190433_1271x259_scrot](https://f.cloud.github.com/assets/376414/1636097/b37d0e46-57c0-11e3-9c2d-e31f229c94e5.png)

That's kind of ugly; we'd like the ribbon on the upper-right, which is where it should be.

If it's too much work, we COULD just have it appear on all the pages, but I think it makes sense mostly to have it on the home page. I'm open to alternative suggestions, though.

```
 <a href="https://github.com/you"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png" alt="Fork me on GitHub"></a>
```

Full selection of ribbons available: https://github.com/blog/273-github-ribbons

**Comment by exchgr** [comment]
What happens if you move the style attribute to the anchor tag?

**Comment by ChimeraCoder** [comment]
Not sure. Try it out and commit it if it works, and you can get your commit
bit for the project page! :-)

Aditya

**Comment by ChimeraCoder** [comment]
Bump @exchgr  - any luck with this?

Hopefully this should be a small/quick fix. We're planning on announcing Pluto/Planet HackNY at the reunion (to those who don't know about it yet), and it'd be great to have your commit in before then if you have time. :)

**Comment by exchgr** [comment]
Hopefully getting to this today.

**Comment by ChimeraCoder** [comment]
Cool, let me know if you need any help

**Comment by ChimeraCoder** [comment]
Closed with db4523fef1cc876e6e8ee661d7394372d58ab58b. Awesome!
